src/main/java/org/jenkinsci/plugins/vsphere/parameters/CloudSelectorParameter.java

```diff
@@ -35,8 +35,10 @@ public StringParameterValue getDefaultParameterValue() {
 
     private StringParameterValue checkValue(StringParameterValue value) {
         List<String> cloudNames = vSphereCloud.findAllVsphereCloudNames();
-        if (!cloudNames.contains(value.value))
-            throw new IllegalArgumentException("No vsphere cloud with name: " + value.value);
+        String check = String.valueOf(value.getValue());
```

Comment:
reason for this change is that `value.value` is illegal to access in recent version of Jenkins.